templates/product_simulator.php

```diff
@@ -104,9 +105,9 @@ function loadSimulatorPagantis()
             simulator_options.itemPromotedAmountSelector = priceSelector;
         }
 
-        if (typeof window.pgSDK != 'undefined') {
+        if (typeof window.pgSDK !== 'undefined') {
             window.WCSimulatorId = window.pgSDK.simulator.init(simulator_options);
-
+            console.trace();
```

Comment:
remove comments

Reply:
The file is not used anymore, so I will remove the file completely